abs.yaml

```diff
@@ -0,0 +1,5 @@
+upload_channels:
+  - sfe1ed40
+channels:
+  - sfe1ed40
+  - https://staging.continuum.io/prefect/fs/umap-learn-feedstock/pr3/8e14b1c
```

Comment:
@Arishamays1 `umap-learn 0.5.4` is now available on `defaults`, therefore you can remove these three lines

```suggestion
```

Reply:
Got it thanks!